routers/web/repo/wiki.go

```diff
@@ -594,6 +594,7 @@ func WikiPages(ctx *context.Context) {
 
 	ctx.Data["Title"] = ctx.Tr("repo.wiki.pages")
 	ctx.Data["CanWriteWiki"] = ctx.Repo.CanWrite(unit.TypeWiki) && !ctx.Repo.Repository.IsArchived
+	ctx.Data["IsRepoOwner"] = ctx.Repo.IsOwner()
```

Comment:
It is already defined in `RepoAssignment` which is called `IsRepositoryOwner`.
And repo admin can also edit settings, so they can also see the default branch in settings page.
So I think `IsRepositoryAdmin` is correct.

Reply:
Thank you, I missed that (I was thinking that wiki might have different repo middlewares ....)